v3/src/hooks/use-measure-text.ts

```diff
@@ -22,3 +26,15 @@ export const useMeasureText = (font = defaultFont) => {
     return measureText(text, font)
   }, [font])
 }
+
+export const findLongestContentWidth = (attr: IAttribute) => {
```

Comment:
With this function here, components in `case-tile-common` import `findLongestContentWidth()` from the `hooks` directory, which then imports `renderAttributeValue()` from the `case-tile-common` directory, so there's a circular reference at the directory level even if there isn't one at the file level. Since the `findLongestContentWidth()` function is specific to the case tiles, I would recommend putting it into the same file with `renderAttributeValue()` and then renaming the file to something like `attribute-format-utils.ts`.